packages/langium/src/parser/langium-parser.ts

```diff
@@ -256,6 +262,17 @@ export class LangiumParser {
         return obj;
     }
 
+    private getAssignment(feature: AbstractElement): AssignmentElement {
+        if (!this.assignmentMap.has(feature)) {
+            const assignment = getContainerOfType(feature, isAssignment);
+            this.assignmentMap.set(feature, {
+                assignment: assignment,
+                crossRef: assignment && isCrossReference(assignment.terminal) || false
```

Comment:
IMO this is more readable: `assignment ? isCrossReference(assignment.terminal) : false`